src/dbseeder/dbseeder.py

```diff
@@ -67,9 +68,15 @@ def seed(self, source, file_location, who):
         programs = self._parse_source_args(source)
 
         for program in programs:
-            seederClass = factory.create(program)
-
-            seeder = seederClass(db, file_location=file_location)
+            seederClass = factory.get(program)
+
+            seeder = seederClass(db=db,
+                                 update=False,
+                                 source=file_location or secrets.sdwis,
```

Comment:
True. But the cli for doing this now is ugly. It doesn't work in powershell because you have to pass empty quotes for file location to sdwis. Maybe the for loop for creating the seeders needs to be smarter.